api/controllers/app/install/apppreflight.go

```diff
@@ -44,6 +44,9 @@ func (c *InstallController) RunAppPreflights(ctx context.Context, opts RunAppPre
 	if err != nil {
 		return fmt.Errorf("extract app preflight spec: %w", err)
 	}
+	if appPreflightSpec == nil {
```

Comment:
Is a preflight spec required? I just saw this support case

https://github.com/replicated-collab/rockwell-automation-replicated/issues/46